### PR TITLE
Add pixelpulsecraft

### PR DESCRIPTION
**Pull request by elxAl33**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Remove directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

background
background@2x
icon
icon@2x
[manifest.json](https://github.com/LabyMod/server-media/files/14073455/manifest.json)

### CODE REVIEW

**Review by elxAl33 — COMMENTED** [review]
New

File: minecraft_servers/pixelpulsecraft/background.png
Comment (maaaathis) [inline minecraft_servers/pixelpulsecraft/background.png:1]: Invalid background size! Size is 1280x653px, must be 1280x720px

File: minecraft_servers/pixelpulsecraft/background@2x.png
Comment (maaaathis) [inline minecraft_servers/pixelpulsecraft/background@2x.png:1]: Invalid hDPI background size! Size is 2117x1080px, must be 1920x1080px

File: minecraft_servers/pixelpulsecraft/icon.png
Comment (maaaathis) [inline minecraft_servers/pixelpulsecraft/icon.png:1]: Invalid icon size! Size is 64x64px, must be 256x256px

File: minecraft_servers/pixelpulsecraft/icon@2x.png
Comment (maaaathis) [inline minecraft_servers/pixelpulsecraft/icon@2x.png:1]: Invalid hDPI icon size! Size is 64x64px, must be 512x512px

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by maaaathis** [comment]
same server as #924, #925, #926, #927 & #928

File: minecraft_servers/pixelpulsecraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "server_wildcards": [
+    "%.PixelPulseCraft.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "twitter": "PixelPulseCraft",
+    "tiktok": "pixelpulsecraft.de",
+    "youtube": "https://www.youtube.com/channel/UCdIaEvVXlF9PtOiabh2UjBA",
```
Comment (maaaathis) [inline minecraft_servers/pixelpulsecraft/manifest.json:14]:
```suggestion
    "youtube": "https://www.youtube.com/channel/UCdIaEvVXlF9PtOiabh2UjBA"

```

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Review by maaaathis — APPROVED** [review]
lgtm

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**